shared/actions/settings.tsx

```diff
@@ -114,7 +114,7 @@ const toggleNotifications = async (state: Container.TypedState) => {
   )
 
   if (!result || !result.body || JSON.parse(result.body)?.status?.code !== 0) {
-    throw new Error(`Invalid response ${result || '(no result)'}`)
+    throw new Error(`Invalid response ${result.body || '(no result)'}`)
```

Comment:
do we need to test `result` not being null/undefined?

```suggestion
    throw new Error(`Invalid response ${result?.body || '(no result)'}`)
```

Reply:
https://github.com/keybase/client/pull/24877